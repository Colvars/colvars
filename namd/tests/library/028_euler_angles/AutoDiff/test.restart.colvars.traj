# step         eulerTheta             eulerPhi               eulerPsi               E_harmonic1            E_harmonic2            E_harmonic3          
          20   -8.80885439249454e-03  -8.09633361337781e-03  -2.04032709423098e-02   2.08146732572652e-05   3.27753089895557e-06   3.87979578540852e-06
          21   -1.11831320841763e-02  -1.00405227961908e-02  -2.29039742557655e-02   2.62296018354385e-05   5.04060490104133e-06   6.25312216060662e-06
          22   -1.35883371105043e-02  -1.21957858042959e-02  -2.55022618552364e-02   3.25182679866522e-05   7.43685956921322e-06   9.23214527143537e-06
          23   -1.59598069397614e-02  -1.42174277267942e-02  -2.79502645085967e-02   3.90608643050260e-05   1.01067625583309e-05   1.27357718777228e-05
          24   -1.82797096501119e-02  -1.63340142360838e-02  -3.04302664834235e-02   4.63000559126085e-05   1.33400010532293e-05   1.67073892446197e-05
          25   -2.04980156243214e-02  -1.80786635495847e-02  -3.26390832270770e-02   5.32654876952029e-05   1.63419037869540e-05   2.10084322267463e-05
          26   -2.26336746042370e-02  -1.97446332702550e-02  -3.48249048090132e-02   6.06386997478414e-05   1.94925271488431e-05   2.56141613045242e-05
          27   -2.46722978973826e-02  -2.08282296097657e-02  -3.66805083826741e-02   6.72729847605713e-05   2.16907574338560e-05   3.04361141768596e-05
          28   -2.66668288111855e-02  -2.16064582471636e-02  -3.84535918821702e-02   7.39339364320254e-05   2.33419518993212e-05   3.55559879422537e-05
          29   -2.86280347132484e-02  -2.16510842256212e-02  -3.98650258568590e-02   7.94610143284018e-05   2.34384724072471e-05   4.09782185771479e-05
          30   -3.11141737511962e-02  -2.11520755490977e-02  -4.14399518801012e-02   8.58634805912551e-05   2.23705150017368e-05   4.84045904109814e-05
          31   -3.36433299652661e-02  -1.99722267367148e-02  -4.26492548570874e-02   9.09479469932396e-05   1.99444920411373e-05   5.65936825575887e-05
          32   -3.62440339256943e-02  -1.83316263303985e-02  -4.36036819876967e-02   9.50640541442094e-05   1.68024261958679e-05   6.56814997603439e-05
          33   -3.89180357713986e-02  -1.63210807958686e-02  -4.42354062542249e-02   9.78385583238159e-05   1.33188839172636e-05   7.57306754151933e-05
          34   -4.16455969878552e-02  -1.40982895480936e-02  -4.45972714321107e-02   9.94458309594679e-05   9.93808840909422e-06   8.67177874237428e-05
          35   -4.44141136081250e-02  -1.20196543655759e-02  -4.47869583592864e-02   1.00293581953823e-04   7.22360455339538e-06   9.86306743797716e-05
          36   -4.71703752676512e-02  -1.01020285878313e-02  -4.48115471628744e-02   1.00403737956526e-04   5.10254907946805e-06   1.11252215144552e-04
          37   -4.98973433183123e-02  -8.83272371876038e-03  -4.49040315400501e-02   1.00818602427491e-04   3.90085041459761e-06   1.24487243511276e-04
          38   -5.25347441121163e-02  -8.03470665045239e-03  -4.50276753744893e-02   1.01374577481520e-04   3.22782554794120e-06   1.37994966946277e-04
          39   -5.50722318522113e-02  -8.14742185048721e-03  -4.54511148078155e-02   1.03290191863661e-04   3.31902414048982e-06   1.51647536059186e-04
          40   -5.79486101479237e-02  -8.88146282109568e-03  -4.61749095421766e-02   1.06606113561410e-04   3.94401909212525e-06   1.67902070903802e-04
