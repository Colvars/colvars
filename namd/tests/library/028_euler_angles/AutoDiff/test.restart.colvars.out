colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "colvars.in":
colvars: # units = "" [default]
colvars: # indexFile = "colvars.ndx"
colvars: The following index groups are currently defined:
colvars:   BFEE_Protein (454 atoms)
colvars:   BFEE_Ligand (75 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 5000 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "eulerTheta"
colvars:   Initializing a new "eulerTheta" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Ligand"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Protein"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 454 atoms requested: total mass = 5981.67, total charge = -65.88.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "reference.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: beginning from 2019-11-26 the XYZ file reader assumes Angstrom units.
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 75 atoms requested: total mass = 980.605, total charge = -9.21.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "reference.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "eulerPhi"
colvars:   Initializing a new "eulerPhi" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Ligand"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Protein"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 454 atoms requested: total mass = 5981.67, total charge = -65.88.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "reference.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 75 atoms requested: total mass = 980.605, total charge = -9.21.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "reference.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "eulerPsi"
colvars:   Initializing a new "eulerPsi" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = on
colvars:       # rotateReference = on
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Ligand"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       Within atom group "atoms":
colvars:       Initializing atom group "fittingGroup".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "BFEE_Protein"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "fittingGroup" defined with 454 atoms requested: total mass = 5981.67, total charge = -65.88.
colvars:       # refPositions =  [default]
colvars:       # refPositionsFile = "reference.xyz"
colvars:       # refPositionsCol = "" [default]
colvars:       Warning: atom group "atoms" will be aligned to a fixed orientation given by the reference positions provided.  If the internal structure of the group changes too much (i.e. its RMSD is comparable to its radius of gyration), the optimal rotation and its gradients may become discontinuous.  If that happens, use fittingGroup (or a different definition for it if already defined) to align the coordinates.
colvars:       # enableFitGradients = on
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 75 atoms requested: total mass = 980.605, total charge = -9.21.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "reference.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { eulerPsi }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 5000 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 0.1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "eulerPsi" will be rescaled to 0.1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic2" [default]
colvars:   # colvars = { eulerPhi }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 5000 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 0.1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "eulerPhi" will be rescaled to 0.1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic3" [default]
colvars:   # colvars = { eulerTheta }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = on
colvars:   # outputFreq = 5000 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 0.1
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "eulerTheta" will be rescaled to 0.1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 3 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (529 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: Re-initialized atom group for variable "eulerTheta":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: Re-initialized atom group for variable "eulerPhi":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: Re-initialized atom group for variable "eulerPsi":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: ----------------------------------------------------------------------
colvars: Loading state from file "test.colvars.state".
colvars:   Restarting collective variable "eulerTheta" from value: -0.00880916
colvars:   Restarting collective variable "eulerPhi" from value: -0.00809521
colvars:   Restarting collective variable "eulerPsi" from value: -0.0204025
colvars:   Restarting harmonic bias "harmonic1" from step number 20.
colvars:   Restarting harmonic bias "harmonic2" from step number 20.
colvars:   Restarting harmonic bias "harmonic3" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "test.restart.restart.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Opening trajectory file "test.restart.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (529 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: Re-initialized atom group for variable "eulerTheta":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: Re-initialized atom group for variable "eulerPhi":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: Re-initialized atom group for variable "eulerPsi":0/0. 75 atoms: total mass = 980.605, total charge = -9.21.
colvars: The restart output state file will be "test.restart.restart.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
